canonical/ubuntu.com
# Implement macros for quote and image section

## Done

- Add macros for quote and image section
- Implemented example use of macros on index page

## QA

- Go to https://ubuntu-com-14149.demos.haus/case-studies
- See that quotes and image section design matches with [Figma](https://www.figma.com/design/AeIJ3GsqnJCHtiBYhkTuTV/24.10-Case-study-template?node-id=1-716&t=ocgEedog4898THvX-0)

## Issue / Card

Fixes [WD-13122](https://warthogs.atlassian.net/browse/WD-13122) and [WD-13123](https://warthogs.atlassian.net/browse/WD-13123)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13122]: https://warthogs.atlassian.net/browse/WD-13122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-13123]: https://warthogs.atlassian.net/browse/WD-13123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
